cpp/src/arrow/python/serialize.cc

```diff
@@ -760,10 +761,14 @@ Status SerializedPyObject::WriteTo(io::OutputStream* dst) {
       dst->Write(reinterpret_cast<const uint8_t*>(&num_buffers), sizeof(int32_t)));
   RETURN_NOT_OK(ipc::WriteRecordBatchStream({this->batch}, dst));
 
+  // Align stream to 64-byte offset so tensor bodies are 64-byte aligned
+  RETURN_NOT_OK(ipc::AlignStream(dst, 64));
```

Comment:
Should these two `64`s be `ipc::kTensorAlignment`?

Reply:
Or vice versa.

Reply:
Yes, I'll change that, and then merge this